### ISSUE TITLE
Alt text not announced in Outlook win32 emails

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14217]
### Steps to reproduce: 
These steps include how to set up an email with this issue.
1. Open Outlook for Windows 
1. Create a new email with ctrl+N
1. Select Insert > Picture with Alt N P2
1. In system insert picture dialog, navigate to and select any image on the device and select OK
1. Open context menu and select View Alt Text
1. Ensure there is alt text in the image
1. Send the email to yourself
1. Open the email from the message list
1. Arrow down to read the image.

### Actual behavior: 
NVDA will read "graphic" but not the alt text added to the image.
NVDA output:
```
Email with an image - NVDA not reading alt text - Message (HTML) 
dialog
document
page 1  section 1  This text is below the image.
This is a test email. I’m adding an image below this line.
down arrow
graphic  
down arrow
This text is below the image.
```

### Expected behavior: 
NVDA will read "graphic" but will not read the text of the alternative text. 

### NVDA logs, crash dumps and other attachments:
```
IO - speech.speech.speak (15:51:59.333) - MainThread (26428):
Speaking [LangChangeCommand ('en_US'), 'This is a test email. I’m adding an image below this line.\r']
IO - inputCore.InputManager.executeGesture (15:52:00.633) - winInputHook (23352):
Input: kb(laptop):downArrow
IO - speech.speech.speak (15:52:00.655) - MainThread (26428):
Speaking [LangChangeCommand ('en_US'), 'down arrow']
IO - speech.speech.speak (15:52:00.811) - MainThread (26428):
Speaking [LangChangeCommand ('en_US'), 'graphic']
DEBUG - UIAHandler.shouldUseUIAInMSWord (15:52:00.944) - MainThread (26428):
Using UIA due to suitable Office version: (16, 0, 15729)
IO - inputCore.InputManager.executeGesture (15:52:01.744) - winInputHook (23352):
Input: kb(laptop):downArrow
IO - speech.speech.speak (15:52:01.755) - MainThread (26428):
Speaking [LangChangeCommand ('en_US'), 'down arrow']
IO - speech.speech.speak (15:52:01.836) - MainThread (26428):
Speaking [LangChangeCommand ('en_US'), 'This text is below the image.']
```

#### NVDA installed/portable/running from source: 
Portable but reproduces on other machines with installed copy.

#### NVDA version: 
2022.3

#### Windows version: 
22H2, OS Build 22623.730

#### Does the issue still occur after restarting your computer? 
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors. 
2022.2 and 2022.3 have the same issue.

#### If NVDA add-ons are disabled, is your problem still occurring? 
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu? 
Yes

**Comment by michaelDCurran** [comment]
I'm unable to reproduce this with NVDA 2022.4 and Outlook for Microsoft 365 MSO (Version 2303 Build 16.0.16227.20002) 64-bit.

Can you test this with a newer version of Outlook please?

After inserting an image, setting alt text, then arrowing up and down the document, NVDA announces the alt text for me.
E.g.:
`graphic a person on a boat`

**Comment by rperez030** [comment]
I can reproduce the problem in this version: 'MSO (Version 2302 Build 16.0.16130.20186) 64-bit'. In fact, I was going to file the issue myself but found this one.

**Comment by Adriani90** [comment]
This is related to the advanced setting "use UIA ofr Microsoft Word documents". This setting is set to default (when suitable) and applies also to MS Outlook, not only to MS Word. When you change the setting to "only necessary", UIA is not primarily used by NVDA in Outlook and then the alt text is being reported as expedcted.
So it seems alt text is not exposed via UIA yet.

I can reproduce this issue with NVDA alpha-27769,872db60c (2023.2.0.27769) on the same Outlook version as @rperez030
@michaelDCurran it seems you tested on a pre-release version of Outlook right?

**Comment by Adriani90** [comment]
I am reopening this until we get a clear picture of the Outlook version in which this issue is solved.

**Comment by Adriani90** [comment]
It seems the alt text is not rendered outside from Windows either. If I send me an email including a grafic with alt text and open the email on my IOS device, the alt text is not reported at all. This needs to be comunicated to Microsoft since it impacts a lot of things such as newsletters, accessible emails etc. I will try to reach out to Microsoft as well, but I guess NV Access has a larger leverage in this regard.

**Comment by Adriani90** [comment]
@msftedad can you please report this to the development team? Actually the alt description seems not to be part of the meta data of a picture, probably that's why it is not read by NVDA when opening an email with alt described images on other devices or in email clients other than Outlook.

Having the alt text of a picture as file meta data (e.g. as meta data of the email source code in case the image is part of the message body) probably would make it easier for other programs to fetch the alt description. But I am not sure, that would need ofcourse some testing.

**Comment by msftedad** [comment]
Hi @Adriani90, We have verified that the issue is not reproducible. Attached is a screenshot for reference. 
<img width="580" alt="image" src="https://github.com/user-attachments/assets/4f33cad6-64a4-4c80-8d46-15eaf012b97b">

**Comment by Adriani90** [comment]
Ah thanks for the update, it seems indeed it is not reproducible anymore in Outlook, no matter whether UIA is on or off.
However, the alt text seems to be recognized by Voice Over on IOS only when opening the email for the first time.
When closing the email window on IOS and opening again, Voice over does not read the alt text anymore. Nonetheless, this is rather related to Voice over and IOS I guess, I am closing this issue as works for me with NVDA.
